doc/concepts/labels.rst

```diff
@@ -98,6 +98,16 @@ the region of cables that have radius less than 0.5 μm
 
 .. _labels-expressions:
 
+.. glossary::
+  iexpr
+    An iexpr is an inhomogeneous expression, that can be evaluated at any point on a cell.
+
+Some common inhomogeneous expression:
+
+* radius: The radius of the cell at a location.
+* distance: Distance to a locset or region from a location.
```

Comment:
Distance here means:
Along the morphology, measured from the closest common ancestor (=minimum distance)